migtests/tests/import-file/run-import-file-test

```diff
@@ -43,7 +43,7 @@ main() {
 
 	step "Import data file: SMSA.txt -> smsa in a non-public schema"
 	import_data_file --data-dir ${TEST_DIR} --format text --delimiter '\t' \
-			--file-table-map "SMSA.txt:smsa"  --start-clean true
```

Comment:
I remember I added `start-clean` explicitly in some of these import file tests to be able to test start-clean for it as we sometime miss import file while making any change to start-clean in import data